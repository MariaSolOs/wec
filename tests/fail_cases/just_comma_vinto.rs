use wec::vinto;

// Make sure the behavior of bec![,] is the same as vec![,].

fn main() {
    let v = vinto![,];
}
