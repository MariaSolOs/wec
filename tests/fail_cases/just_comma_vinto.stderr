error: no rules expected the token `,`
 --> tests/fail_cases/just_comma_vinto.rs:6:20
  |
6 |     let v = vinto![,];
  |                    ^ no rules expected this token in macro call
  |
  = note: while trying to match sequence start
